Lib/test/test_email/test_message.py

```diff
@@ -784,6 +784,137 @@ def test_str_defaults_to_utf8(self):
         m['Subject'] = 'unicöde'
         self.assertEqual(str(m), 'Subject: unicöde\n\n')
 
+    def test_folding_with_utf8_encoding_1(self):
+        # issue #36520
```

Comment:
Shouldn't this say "bpo-36520"?  What would "issue #36520" refer to if we move to GitHub issues?

Reply:
I have made the requested changes; please review again